README.md

```diff
@@ -581,6 +581,23 @@ end
 return M
 ```
 
+To enable TLS, provide the following params into roles config (for proper work
+it's enough to provide only `ssl_key_file` and `ssl_cert_file`):
+
+```yaml
+roles_cfg:
+    roles.httpd:
+        default:
+            listen: 8081
+            params:
+              ssl_key_file: "path/to/key/file"
+              ssl_cert_file: "path/to/key/file"
+              ssl_ca_file: "path/to/key/file"
+              ssl_ciphers: "cipher1:cipher2"
+              ssl_password: "password"
+              ssl_password_file: "path/to/ssl/password"
```

Comment:
If you want to be similar to tarantool settings:
```suggestion
            listen:
              uri: 8081
              params:
                ssl_key_file: "path/to/key/file"
                ssl_cert_file: "path/to/key/file"
                ssl_ca_file: "path/to/key/file"
                ssl_ciphers: "cipher1:cipher2"
                ssl_password: "password"
                ssl_password_file: "path/to/ssl/password"
```

Reply:
Apply the next suggestion.